src/m_menu.c

```diff
@@ -3307,12 +3307,11 @@ setup_menu_t stat_settings1[] =  // Status Bar and HUD Settings screen
   {"GRAY %"            ,S_YESNO|S_COSMETIC,m_null,M_X,M_Y+stat1_graypcnt*M_SPC, {"sts_pct_always_gray"}},
   {"SINGLE KEY DISPLAY",S_YESNO|S_COSMETIC,m_null,M_X,M_Y+stat1_keys*M_SPC,     {"sts_traditional_keys"}},
 
-  {"",S_SKIP,m_null,M_X,M_Y+stat1_stub1*M_SPC},
-
```

Comment:
I figured that this page was the most fitting for the new menu item, but I have to remove this stub to leave space for the "Value out of range" warning, which makes the whole menu look more cramped. Let me know if you want me to move the new item elsewhere.

Reply:
It could fit better on the second page if we rename the "Widget Colors" paragraph to something more general, e.g. "Widget Appearance".

Reply:
Sounds good to me. Unless you'd like to merge the PR then move the new item yourself, let me know once you settle for a new name for said paragraph and I'll do the changes myself.

Reply:
Yes, please do as I suggested. 😉

Reply:
So, if you could change the menu as suggested, I'd merge this.